lib/modules/showImages.js

```diff
@@ -488,30 +489,22 @@ module.beforeLoad = () => {
 		`);
 	}
 
-	watchForElements(['siteTable'], getSelector(), checkElementForMedia);
-	watchForElements(['selfText', 'newComments'], getSelector(true), checkElementForMedia);
-};
+	watchForElements(['selfText'], null, scanBody);
+	watchForThings(['comment', 'message'], thing => scanBody(thing.getTextBody()));
```

Comment:
May as well add `post` in here, so RES can show selftext expandos in lightbox and comments full-page views.  

I'll make a note that the redesign JSAPI add a property like "expando showing" which is updated when self-text expandos are opened.